### PR TITLE
Add container=oci to Dockerfile

### PR DESCRIPTION
**Pull request by spadgett**
/assign @adammhaile

### CODE REVIEW

**Comment by spadgett** [comment]
@adammhaile PTAL

File: images/origin-web-console/Dockerfile [hunk 1]
```diff
@@ -2,10 +2,12 @@
 # This is the integrated Origin Web Console Server. It is configured to
 # publish metadata to OpenShift to provide automatic management of images on push.
 #
-# The standard name for this image is openshift/origin-docker-registry
+# The standard name for this image is openshift/origin-web-console
 #
 FROM openshift/origin-base
 
+ENV container=oci
```
Comment (adammhaile) [inline images/origin-web-console/Dockerfile:9]: I guess this change is fine? What's this line for... doesn't seem used.

**Pull request closed without merging**